examples/community/pipeline_faithdiff_stable_diffusion_xl.py

```diff
@@ -1705,6 +1705,12 @@ def enable_vae_tiling(self):
         compute decoding and encoding in several steps. This is useful for saving a large amount of memory and to allow
         processing larger images.
         """
+        depr_message = "Calling `enable_vae_tiling()` on a `DiffusionPipeline` is deprecated and this method will be removed in a future version. Please use `pipe.vae.enable_tiling()`."
```

Comment:
Perhaps use "Calling `enable_vae_tiling()` on a `{cls.__name__}` is deprecated`? So the specific pipeline name shows up in the message?